README.md

```diff
@@ -1,2 +1,76 @@
-# github-burndown
-Generate burndown chart from GitHub issues, via Google Sheets
+# GitHub Burndown Chart Generator
+
+## Objective:
+The GitHub Burndown Chart Generator aims to create a burndown chart from GitHub issues using Google Sheets through the power of Apps Script.
+
+## How to Run:
+
```

Comment:
How about if I share my Google Sheet, then people just do `Make a copy`?

Reply:
I think we still need to add the part to update GITHUB_TOKEN, but other than that I agree. I will remove this